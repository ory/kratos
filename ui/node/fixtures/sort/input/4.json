[
  {
    "attributes": {
      "disabled": false,
      "name": "csrf_token",
      "required": true,
      "type": "hidden",
      "value": ""
    },
    "group": "default",
    "messages": [],
    "meta": {},
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "traits.email",
      "type": "email",
      "value": "john@doe.com"
    },
    "group": "profile",
    "messages": [],
    "meta": {},
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "traits.name",
      "type": "text"
    },
    "group": "profile",
    "messages": [],
    "meta": {},
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "method",
      "type": "submit",
      "value": "profile"
    },
    "group": "profile",
    "messages": [],
    "meta": {
      "label": {
        "id": 1070003,
        "text": "Save",
        "type": "info"
      }
    },
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "password",
      "required": true,
      "type": "password"
    },
    "group": "password",
    "messages": [],
    "meta": {
      "label": {
        "id": 1070001,
        "text": "Password",
        "type": "info"
      }
    },
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "method",
      "type": "submit",
      "value": "password"
    },
    "group": "password",
    "messages": [],
    "meta": {
      "label": {
        "id": 1070003,
        "text": "Save",
        "type": "info"
      }
    },
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "link",
      "type": "submit",
      "value": "github"
    },
    "group": "oidc",
    "messages": [],
    "meta": {
      "label": {
        "context": {
          "provider": "github"
        },
        "id": 1050002,
        "text": "Link github",
        "type": "info"
      }
    },
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "unlink",
      "type": "submit",
      "value": "google"
    },
    "group": "oidc",
    "messages": [],
    "meta": {
      "label": {
        "context": {
          "provider": "google"
        },
        "id": 1050003,
        "text": "Unlink google",
        "type": "info"
      }
    },
    "type": "input"
  },
  {
    "attributes": {
      "disabled": false,
      "name": "link",
      "type": "submit",
      "value": "ory"
    },
    "group": "oidc",
    "messages": [],
    "meta": {
      "label": {
        "context": {
          "provider": "ory"
        },
        "id": 1050002,
        "text": "Link ory",
        "type": "info"
      }
    },
    "type": "input"
  }
]
