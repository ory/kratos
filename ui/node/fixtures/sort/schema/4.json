{
  "$id": "https://example.com/person.schema.json",
  "$schema": "http://json-schema.org/draft-07/schema#",
  "title": "Person",
  "type": "object",
  "properties": {
    "traits": {
      "type": "object",
      "properties": {
        "email": {
          "format": "email",
          "type": "string",
          "ory.sh/kratos": {
            "credentials": {
              "password": {
                "identifier": true
              }
            }
          }
        },
        "name": {
          "type": "string",
          "minLength": 2
        }
      },
      "required": [
        "email"
      ]
    }
  },
  "additionalProperties": false
}
