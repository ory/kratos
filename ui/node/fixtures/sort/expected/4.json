[
  {
    "type": "input",
    "group": "default",
    "attributes": {
      "name": "csrf_token",
      "type": "hidden",
      "value": "",
      "required": true,
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "profile",
    "attributes": {
      "name": "traits.email",
      "type": "email",
      "value": "john@doe.com",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "profile",
    "attributes": {
      "name": "traits.name",
      "type": "text",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {}
  },
  {
    "type": "input",
    "group": "profile",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "profile",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1070003,
        "text": "Save",
        "type": "info"
      }
    }
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "password",
      "type": "password",
      "required": true,
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1070001,
        "text": "Password",
        "type": "info"
      }
    }
  },
  {
    "type": "input",
    "group": "password",
    "attributes": {
      "name": "method",
      "type": "submit",
      "value": "password",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1070003,
        "text": "Save",
        "type": "info"
      }
    }
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "link",
      "type": "submit",
      "value": "github",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1050002,
        "text": "Link github",
        "type": "info",
        "context": {
          "provider": "github"
        }
      }
    }
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "unlink",
      "type": "submit",
      "value": "google",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1050003,
        "text": "Unlink google",
        "type": "info",
        "context": {
          "provider": "google"
        }
      }
    }
  },
  {
    "type": "input",
    "group": "oidc",
    "attributes": {
      "name": "link",
      "type": "submit",
      "value": "ory",
      "disabled": false,
      "node_type": "input"
    },
    "messages": [],
    "meta": {
      "label": {
        "id": 1050002,
        "text": "Link ory",
        "type": "info",
        "context": {
          "provider": "ory"
        }
      }
    }
  }
]
